### DISCUSSION TITLE
switch cases may not be coverable

### DISCUSSION THREAD

**Discussion by SYNTEC-10101867** [FortuneN/FineCodeCoverage#340] [Q&A]
Hello,
I met a problem on missing coverable lines when there are switch cases more than five cases in my code, but it seems work fine if my switch cases contain only five cases or less. 

Here is my working environment :
vs2022 c++ project
googletest

Thanks

**Comment by tonyhallett** [comment]
Please provide a minimal solution that exhibits this behaviour.
